### ISSUE TITLE
No effect on Silver iMac

### ISSUE THREAD

**Issue by EclipsedVisions** [elunico/iMacColorUnlocker#1]
Hello,
Wanted to try that app out but the accent colors do not change for me. I tried printing out the variables using terminal and they do seem alright. I'm on the M1 iMac in silver running macOS Ventura

**Comment by elunico** [comment]
I would like to try and help you but I do not have an iMac to test on. Still, I will look into it. 

Can you provide be the exact version number and build of the macOS you are running?

Also please note also that you **must restart** open apps to see the changes. This might mean restarting Settings after choosing the color in the application itself as well as logging out or restarting once you select the color you would like

**Comment by EclipsedVisions** [comment]
Yeah, i've tried logging out then logging back in, restarting, killing the finder process and restarting settings, however i didn't get the extra color selection in the settings no matter what. As for the macOS build, i'm on version 13.4.1 (22F82). As far as i understood this only happens to people on the silver imac which is a bit weird imo. could be missing the accent bundles or something?

**Comment by elunico** [comment]
Unfortunately, I am unable to reproduce this issue on any of the 3 Macs I test on including 2 M1 Macs and an intel machine. However, I am not able to access an M1 iMac for the purpose of testing. It is possible that there is something in the SIP keeping the settings from changing since your computer is an iMac already, but I am not sure. I am open to a solution if it appears, but I cannot actively seek to fix this problem since I cannot reproduce it.
